lxc/remote.go

```diff
@@ -402,8 +405,9 @@ func (c *cmdRemoteGetDefault) Run(cmd *cobra.Command, args []string) error {
 
 // List
 type cmdRemoteList struct {
-	global *cmdGlobal
-	remote *cmdRemote
+	global     *cmdGlobal
+	remote     *cmdRemote
+	flagFormat string
```

Comment:
Add a separate blank line before flagFormat, I believe we do that everywhere else